custom.vite.config.mjs

```diff
@@ -0,0 +1,50 @@
+import { defineConfig } from 'vite';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
+import * as path from 'path';
+import sveltePreprocess from 'svelte-preprocess';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  resolve: {
+    alias: [
+      {
+        find: '@nuclia/ui',
+        replacement: path.resolve(__dirname, './libs/nuclia/libs/search-widget/src/index.js'),
+      },
+      {
+        find: '@nuclia/core',
+        replacement: path.resolve(__dirname, './libs/nuclia/libs/sdk-core/src/index.ts'),
+      },
+      {
+        find: '@nuclia/prediction',
+        replacement: path.resolve(__dirname, './libs/nuclia/libs/prediction/src/index.ts'),
+      },
+    ],
+  },
+  build: {
+    outDir: `dist/my-widget`,
```

Comment:
in the readme you explained `my-widget` was just you example choice, but you didn't explain that by changing the widget name, the user would need to also update this vite config or create their own accordingly.